### ISSUE TITLE
Devices and Sensors WG - 2021 Q2 virtual meeting

### ISSUE THREAD

**Issue by anssiko** [w3c/devicesensors-wg#39]
# Devices and Sensors WG - 2021 Q2 virtual meeting

Please find joining instructions (login first), agendas, and invites from the W3C Calendaring tool:

🆕 [New work](https://www.w3.org/events/meetings/756f098f-08f0-4230-80fb-4762754996a9)
⏰ April 06 2021, 05:00 - 06:00 UTC
[Meeting minutes](https://www.w3.org/2021/04/06-dap-minutes.html)

🔋 [Other device capabilities](https://www.w3.org/events/meetings/a8a1448f-049b-468e-8b45-8eae6b8e75be)
⏰ April 06 2021, 15:00 - 17:00 UTC
[Meeting minutes](https://www.w3.org/2021/04/06-das-minutes.html)

🔒 [Privacy and architecture](https://www.w3.org/events/meetings/947245c6-b7eb-4b38-9a34-358d19f97c9d)
⏰ April 07 2021, 05:00 - 07:00 UTC
[Meeting minutes](https://www.w3.org/2021/04/07-dap-minutes.html)

📡 [Sensors](https://www.w3.org/events/meetings/25801de1-caf6-434c-b20c-6ed1819cbe90)
⏰ April 08 2021, 05:00 - 07:00 UTC
[Meeting minutes](https://www.w3.org/2021/04/08-dap-minutes.html)

**Comment by willmorgan** [comment]
I will attend to "sponsor" the screen brightness API, whatever time works 👍

**Comment by anssiko** [comment]
@willmorgan thanks! I just tagged your name next to that topic in the agenda. This is also an API candidate to be brought into the WG once sufficiently incubated first, so noted that in New work section as well.

**Comment by anssiko** [comment]
# What you should know

Chime in on this issue if you have a scheduling conflict. Some usual suspects to drive discussions have been identified below. Additional agenda proposals are welcome.

Please follow the links in the OP for the latest. We'll push another update to your calendars when the schedule and agenda stabilizes.

Thank you,

Chairs @anssiko @reillyeon and Team Contact @xfq 

# Proposed meeting schedule

🆕 New work
⏰ April 06 2021, 05:00 - 06:00 UTC

🔋 Other device capabilities
⏰ April 06 2021, 15:00 - 17:00 UTC

🔒 Privacy and architecture
⏰ April 07 2021, 05:00 - 07:00 UTC

📡 Sensors
⏰ April 08 2021, 05:00 - 07:00 UTC

# Proposed agenda

### 🆕 New work

- WICG incubations of interest
   - Contact Picker API @xfq @marcoscaceres
      - Proposed for DAS WG https://github.com/w3c/dap-charter/issues/97
   - Screen Brightness API @willmorgan
      - Discuss WICG proposal feedback and readiness https://github.com/WICG/proposals/issues/17
- Other Proposals for discussion
  - User Presence API @anssiko
      - Gauge interest for this new capability, standalone and/or an extension to existing Proximity @anssiko

### 🔋 Other device capabilities

- Screen Fold API -> Device Posture API @kenchris
  - New tighter scope on device posture
- Screen Wake Lock API @rakuco
  - Recent updates to modernize the spec language e.g. https://github.com/w3c/screen-wake-lock/pull/299
  - Auto-releasing wake lock https://github.com/w3c/screen-wake-lock/issues/300
- System Wake Lock API @reillyeon
  - Follow up on [TPAC resolution](https://www.w3.org/2020/10/22-dap-minutes.html#r02) to write an explainer
- Battery Status API @anssiko
  - [SecureContext] migration https://github.com/w3c/battery/issues/15
  - Changes to align with the implementation https://github.com/w3c/battery/pull/30
- Geolocation API @marcoscaceres
   - TAG review changes https://github.com/w3ctag/design-reviews/issues/529
   - Privacy enhancement https://github.com/w3c/geolocation-api/pull/64
   - Explicitly limit permission lifetimes https://github.com/w3c/geolocation-api/issues/47
   - Path to REC https://github.com/w3c/geolocation-api/issues/42
- Network Information API
  - Follow up on [TPAC resolution](https://www.w3.org/2020/10/22-dap-minutes.html#r01) to revise use cases for 2020
- HTML Media Capture @anssiko
  - Integration into HTML Living Standard per [W3C/WHATWG Update](https://www.w3.org/2021/03/WHATWG-W3C-MOU.html)

### 🔒 Privacy and architecture

- Incorporating "AB/TAG experiment" recommendations to specs
  - Review and discuss issues filed in response to this experiment, many privacy related
- Privacy cross-group collaborations
  - Updates from Privacy CG and PING relevant to the DAS WG
- TAG collaboration
  - Refresh of TAG reviews for specs that have not been reviewed in a while
  - Discuss any TAG design guidelines relevant to the work in scope, e.g. https://github.com/w3ctag/design-principles/issues/39

### 📡 Sensors

- Accelerometer @rakuco
  - GravitySensor spec refinements, explainer updates
- Proximity @anssiko
  - Ecma TC53 interop reqs, e.g. https://github.com/w3c/proximity/issues/44
  - "near" and "user presence state" interaction https://github.com/w3c/screen-wake-lock/issues/300
- Ambient Light Sensor @willmorgan
  - Screen Brightness API spin-off https://github.com/w3c/ambient-light/issues/64 https://github.com/WICG/proposals/issues/17 @willmorgan

**Comment by anssiko** [comment]
Here’s the latest virtual meeting logistics update: https://lists.w3.org/Archives/Public/public-device-apis/2021Apr/0000.html

Notably:

> If you don't see the Joining Instructions on the above pages that contain Zoom coordinates, please chime in on IRC http://irc.w3.org/?channels=#dap [note: http(!)] at the beginning of each session and we'll let you in.

**Comment by anssiko** [comment]
FTR, minutes shared on the mailing list: https://lists.w3.org/Archives/Public/public-device-apis/2021Apr/0007.html

Also updated the OP with links to the minutes.

**Comment by rakuco** [comment]
@anssiko the event pages in w3.org/events/meetings are linking to the wrong minutes:
* [Other device capabilities](https://www.w3.org/events/meetings/a8a1448f-049b-468e-8b45-8eae6b8e75be) is linking to the New Work minutes.
* [Privacy and architecture](https://www.w3.org/events/meetings/947245c6-b7eb-4b38-9a34-358d19f97c9d) is linking to a URL that does not exist

**Comment by anssiko** [comment]
@rakuco thanks for reporting, the URLs are now fixed.

On this topic, if folks have feedback for the W3C Calendar tool we just beta tested for this meeting, feedback is welcome.

I have couple of feature requests on my mind:

- Add links to the minutes directly to this listing: https://www.w3.org/groups/wg/das/calendar?past=1&tf=0 to avoid the need to click "More information" > Minutes.

- Default to "View Past Events" if there are no planned upcoming events. Otherwise https://www.w3.org/groups/wg/das/calendar is not so useful.

I already opened an issue for the "invite changed spam" you received and there's a fix to that underway, see https://github.com/w3c/calendar/issues/24

If many folks think the Calendar tool made things worse, then let us know are we can revert to the manual way of doing meetings where you get to manage your own calendars manually.
